src/routes/cheatsheet/cheat-sheet.js

```diff
@@ -437,6 +437,8 @@ onMount(() => {
     alert('count is dangerously high!')
     count = 9
   }
+  
+  $: ({ab, cd, ef: {gh} = {gh: {}}} = alphabet)
```

Comment:
It's a bit hard to understand what you trying to do.
And furthermore, the code is "invalid": `alphabet` is undefined

Something like that seem easier to understand:
```suggestion
  let myObject = {a: 10, b: 20, c: { d: "hello" }}
  $: ({a, b, c} = myObject)
```

Reply:
I wanted to make this code as difficult as possible to explain well what is possible to do.

Maybe the below is better?

```ts
let myObject = {a: 10, b: 20, c: { d: "hello" }}

$: ({a, b, c, e: {f} = {f: "default"}} = myObject)
```

Reply:
I'm not sure if there are any themes being used in the cheatsheet, but one device I've seen work well is to [explain things with super heroes](https://dev.to/techsquidtv/combine-objects-in-typescript-3f64). Otherwise, finding something with short identifiers (e.g. coordinates) might be more useful than asking the reader to assign identifiers meaning on the fly while trying to grok the concept.